README.md

```diff
@@ -142,13 +168,13 @@ Bug reports, enhancement suggestions, and pull requests are welcome at the [Obsi
 Here are some possible enhancements already contemplated.
 
 - List details of vault plugins, snippets, config.
-- More fine-grained control over which files and directories are copied, and which destination vaults they are copied to.
+- More fine-grained control over which files and directories are copied, and which destination vaults they are copied to. This would replace the `ITEMS_TO_COPY` global.
```

Comment:
Thinking about this, probably have a config file somewhere to get that list completely out of the code.